src/components/common/Home/CreateTargetForm/create-target-form.scss

```diff
@@ -5,6 +5,11 @@
   align-items: center;
 }
 
+.new-target-container {
+  @extend .new-target;
+  width: 35%;
+}
+
```

Comment:
`.extend` will work, but as `.new-target` definition is just above, I think it's simpler to just add it to the rules declaration like this:
```scss
.new-target,
.new-target-container {
  .....
}

.new-target-container {
  width: 35%;
}

```

Reply:
yes, that seems better!